src/config/models.py

```diff
@@ -34,10 +34,10 @@ class HandleRaresType(enum.StrEnum):
 
 
 class MoveItemsType(enum.StrEnum):
+    favorites = enum.auto()
     everything = enum.auto()
```

Comment:
Move favorites down one more, it should be after everything to keep it alphabetical. I'd do it myself but I don't think I can